### PR TITLE
Add updated_at to plugins/plugin configs

### PR DESCRIPTION
**Pull request by macobo**
This will be used in plugin server to calculate which plugins have changed

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

File: posthog/migrations/0131_add_plugins_updated_created_at.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            model_name="plugin",
+            name="created_at",
+            field=models.DateTimeField(auto_now_add=True, default=datetime.datetime(2020, 1, 1, 0, 0)),
+            preserve_default=False,
+        ),
+        migrations.AddField(model_name="plugin", name="updated_at", field=models.DateTimeField(auto_now=True),),
+        migrations.AddField(
+            model_name="pluginconfig",
+            name="created_at",
+            field=models.DateTimeField(auto_now_add=True, default=datetime.datetime(2020, 1, 1, 0, 0)),
```
Comment (Twixes) [inline posthog/migrations/0131_add_plugins_updated_created_at.py:25]: These defaults look weirdly hardcoded, shouldn't they be `django.utils.timezone.now`?

Comment (macobo) [reply posthog/migrations/0131_add_plugins_updated_created_at.py:25]: Good question. I left a weird default so we're able to distinguish "old" plugins vs new ones after this PR. Having .now() would be misleading whenever looking at the db.

**Review by Twixes — APPROVED** [review]

**Comment by yakkomajuri** [comment]
This broke tests on the plugin server:

```
error: null value in column "created_at" of relation "posthog_plugin" violates not-null constraint
```

**Comment by Twixes** [comment]
Oh right, v. easy to sync up though. Did you do that in any plugin server PR already or can I make one?

**Comment by yakkomajuri** [comment]
I posted this before investigating further - haven't put out a PR to fix, feel free to.

**Pull request merged**